src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Sfa.Das.Sas.Core.Configuration;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.SaveApprenticeship;
+using System;
+using System.Web;
+
+namespace Sfa.Das.Sas.Shared.Components.ViewComponents.Basket.SaveApprenticeship
+{
+    public class SaveApprenticeshipViewComponent : ViewComponent
+    {
+        private readonly IFatConfigurationSettings _config;
+
+        public SaveApprenticeshipViewComponent(IFatConfigurationSettings config)
+        {
+            _config = config;
+        }
+
+        public IViewComponentResult Invoke(string apprenticeshipId)
+        {
+            var saveUrl = new Uri(new Uri(_config.EmployerFavouritesUrl), "/save-apprenticeship-favourites");
```

Comment:
should the save url be part of config along with the url?

Reply:
Yeah I was toying with this one. Will move it to the config.